README.md

```diff
@@ -180,7 +176,7 @@ Here are some features in progress or planned: https://github.com/gear-tech/gear
 
 # License
 
-Gear is licensed under [GPL v3.0 with a classpath linking exception](LICENSE).
+Gear Protocol is licensed under [GPL v3.0 with a classpath linking exception](LICENSE).
```

Comment:
and Vara Network maybe?

Reply:
How can we license the network? Only source code may be under some license, I guess.